application-workloads/dosyago/browserbox/notifier.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
I'm not following where this script is being called. Can you share what I am missing?